app/src/test/java/com/futurice/freesound/functional/StringFunctionsTest.java

```diff
@@ -23,33 +23,33 @@
 public class StringFunctionsTest {
 
     @Test
-    public void isEmpty_returnsTrue_whenCalled_withEmptyString() {
-        assertThat(StringFunctions.isEmpty().call("")).isTrue();
+    public void isEmpty_returnsTrue_whenCalled_withEmptyString() throws Exception {
+        assertThat(StringFunctions.isEmpty().test("")).isTrue();
```

Comment:
No, we can't rename it, so no need to change it. I was just speculating if they might (or perhaps they will change the `test` method on `Observable`.